templates/components/cart/content.html

```diff
@@ -25,7 +25,42 @@
                             {{name}}
                         </a>
                         <div class="c-cart__item-rating-wrapper">
-                            <span class="c-cart__item-rating" role="img" aria-label=""></span>
+                            <div class="c-cart__item-rating" role="img" aria-label="">
+                                <span class="c-cart-item__filled-stars">
+                                    <svg class="u-hidden" xmlns="http://www.w3.org/2000/svg" width="12" height="12" viewBox="0 0 12 12" fill="none">
```

Comment:
Everytime you repeat the same code, the red light should appear above your head saying "there must be a better way to approach this" :D 

Please create a separate component for the star, where you could pass the value "filled" or "unfilled"  and you could either use SVG sprite (https://github.com/xfiveco/xcorner?tab=readme-ov-file#icons) and toggle visibility similar as you do now. Or even better, you could pass the rating value into the "product rating" component and pass the rating then display star svg + clip path to present the exact rate value. 

Example for the single star: 
```
<svg>
  <defs>
    <clipPath id="star">
      <path d="...." />
    </clipPath>
  </defs>

  <rect style="width: var(--width, 100%); fill: var(--color, #dd2424)" clip-path="url(#star)"/>
</svg>
```